pyccel/codegen/printing/ccode.py

```diff
@@ -488,36 +486,56 @@ def _print_PythonLen(self, expr):
                     PYCCEL_RESTRICTION_TODO,
                     symbol = expr, severity='fatal')
 
+    def _print_Header(self, expr):
+        return ''
+
     def _print_ModuleHeader(self, expr):
         name = expr.module.name
         # TODO: Add classes and interfaces
         funcs = '\n'.join('{};'.format(self.function_signature(f)) for f in expr.module.funcs)
 
+        variables = ''.join(['extern '+self._print(d) for d in expr.module.declarations if not d.variable.is_private])
```

Comment:
I think `global_vars` may be a meaningful name than `varaible`.

Reply:
Fixed 8e7cc1c0